docs/docsource/reference/command_reference/s3api_commands/get-bucket-notification-configuration.rst.txt

```diff
@@ -64,7 +64,7 @@ QueueConfigurations -> (list)
   (structure)
 
     Specifies the configuration for publishing messages to a Kafka queue w`hen
-    Zenko detects specified events.
+    XDM  detects specified events.
```

Comment:
~~XDM~~ -> |product| ?

Reply:
Looks like I've got more fun with sed ahead...

Reply:
The "rst.txt" files, thankfully, do not build and are not noticed by Sphinx. This one, and one other (put_bucket_notification_configuration.rst.txt) are APIs that I built the documentation for, but the feature was either withdrawn or sidelined. I'm storing them here because it maintains context without breaking the build. I assume my script went after "*.rst" and missed these. I'll fix them anyway.